app/controllers/forms_controller.rb

```diff
@@ -43,6 +43,18 @@ def update
     end
   end
 
+  def new
+    form = Form.create! name: 'new-form'
```

Comment:
Right, I mean, my question was just because I'm curious how clear it is what the name even does.